vllm/model_executor/layers/quantization/gptq_marlin.py

```diff
@@ -62,11 +70,36 @@ def __init__(
 
         self.quant_type = self.TYPE_MAP[(weight_bits, is_sym)]
 
+    def update_config(self, prefix: str):
```

Comment:
This seems like a bit of a hack at the moment, so please leave a comment on why this is needed and what case it is used. It might be good to generally document that `dynamic_cfg` comes from the GPTQModel spec

Reply:
Method renamed to `override_config` to be a little more useful/accurate on what it does plus extensive comments on what `dynamic_cfg` is.